tests/unit/test_sqlite_graph.py

```
@@ -46,13 +48,15 @@ async def test_create_node_basic(self, adapter: SQLiteGraphAdapter) -> None:
         node_ids = [n["id"] for n in result.nodes]
         assert "node-1" in node_ids
 
+    @pytest.mark.asyncio
     async def test_create_node_idempotent(self, adapter: SQLiteGraphAdapter) -> None:
         """同じノードIDで2回呼び出しても例外が発生しない（upsert）."""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `ノ` (KATAKANA LETTER NO). Did you mean `/` (SOLIDUS)?

(RUF002)